common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -1,17 +1,38 @@
 package uk.ac.wellcome.models.work.internal
 
-sealed trait BaseImage[+Id] extends HasIdState[Id] {
+sealed trait BaseImage[+Id <: IdState] extends HasIdState[Id] {
   val id: Id
   val location: DigitalLocation
 }
 
-case class UnmergedImage[+Id](id: Id, location: DigitalLocation)
-    extends BaseImage[Id]
+case class UnmergedImage[Id <: IdState](
+  id: Id,
+  location: DigitalLocation
+) extends BaseImage[Id] {
+  def mergeWith(data: => ImageData[Id]): MergedImage[Id] =
+    MergedImage[Id](
+      id = id,
+      location = location,
+      data = data
+    )
+}
 
-case class MergedImage[+Id](id: Id, location: DigitalLocation, data: ImageData)
-    extends BaseImage[Id]
+case class MergedImage[Id <: IdState](
+  id: Id,
+  location: DigitalLocation,
+  data: ImageData[Id]
+) extends BaseImage[Id] {
+  def toUnmerged: UnmergedImage[Id] =
+    UnmergedImage[Id](
+      id = id,
+      location = location
+    )
+}
 
-case class ImageData(title: Option[String], parentWorks: List[String])
+case class ImageData[Id <: IdState](
+  parentWork: Id,
+  fullText: Option[String] = None
+)
```

Comment:
Is having a seperate `ImageData` class necessary? The reason we have `WorkData` is that multiple work types need the same data, but here it appears like only one does

Reply:
Yea, I'd say of we can avoid this nesting in the model, it would be nice.

Reply:
Yeah, I'll have a go at removing it - don't think it is necessary